pkg/pillar/uuidtonum/uuidstonum.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2018 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package uuidtonum
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/lf-edge/eve/pkg/pillar/base"
+	"github.com/lf-edge/eve/pkg/pillar/pubsub"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"github.com/satori/go.uuid"
+)
+
+// UUIDs to  Number Map Routines
+func uuidsToNumKey(baseID uuid.UUID, appID uuid.UUID) string {
```

Comment:
The added "s" in the name is probably to subtle. How about naming the file and the functions "uuidPair"?

Reply:
done